src/libs/ui/docsetsdialog.cpp

```diff
@@ -675,6 +675,15 @@ bool DocsetsDialog::updatesAvailable() const
 QNetworkReply *DocsetsDialog::download(const QUrl &url)
 {
     QNetworkReply *reply = m_application->download(url);
+
+    if (m_application->settings()->isIgnoreSSLErrorsEnabled) {
+        // Connect to the reply's sslErrors signal to handle SSL errors
+        connect(reply, &QNetworkReply::sslErrors, [=](const QList<QSslError>& errors){
+            // Ignore all SSL errors
+            reply->ignoreSslErrors();
+        });
+    }
```

Comment:
This signal should be handled on [`Core::NetworkAccessManager`](https://github.com/zealdocs/zeal/blob/94247fd4d03e6ae079cae8e50e6eb9a67fc0afad/src/libs/core/networkaccessmanager.h) to cover all network activity performed by Zeal.

The easiest approach would probably be handling the signal in `Core::Application`, when network manager is [created](https://github.com/zealdocs/zeal/blob/94247fd4d03e6ae079cae8e50e6eb9a67fc0afad/src/libs/core/application.cpp#L64). Not the most elegant solution, but would avoid the need to pass settings into `Core::NetworkAccessManager`.

Reply:
You meant like this - https://github.com/zealdocs/zeal/pull/1521/commits/b4e2c86816c435e0497030d284eccc0243c32b74 ?